charts/refinery/templates/ingress-beta.yaml

```diff
@@ -26,8 +26,8 @@ spec:
       secretName: {{ .secretName }}
     {{- end }}
   {{- end }}
-  {{- if .Values.ingress.className }}
-  ingressClassName: {{ .Values.ingress.className }}
+  {{- if or .Values.ingress.ingressClassName .Values.ingressClassName }}
+  ingressClassName: {{- if or .Values.ingress.ingressClassName .Values.ingressClassName }}
```

Comment:
This is not a valid use of helm's `if` flow control.  When I checkout this branch and run `helm template refinery ./charts/refinery --set ingressClassName=nginx --set ingress.enabled=true` I get helm template errors.  I think instead we want to use [helm's default function](https://helm.sh/docs/chart_template_guide/function_list/#default).